### ISSUE TITLE
M134 Mingun Event gun

### ISSUE THREAD

**Issue by Haox09** [deferred-impact/battledudes-issue-tracker#125]
M134 is a event only item, apearing only in the month of April. Starter ammo it have 400, 200 already loaded so the ammo count will show 200/200. reload time 5 seconds. Extended mag perk won't work on it but full pocket does. Ammo can be refilled from ammo box, but slower than most guns. The shooting have the accuracy of m1, dmg is under 10. Now to the special shooting. Delayed trigger: when left click it delays 1 second before shooting very fast. Comparable to a smg. When shooting the player won't be able to move, and when not shooting can move in the speed of 1.5 blocks per second. The switch speed from other weapon will be 4 seconds, quick switch perk works on it. When equipped in the loadout, the player can't have a secondary. Also overkill perk doesn't work on M134. 
The M134 should work best on big area enemy killing. High accuracy makes it weak close up. The no movement and no seconddary also makes it vulnerable close up. 
To unlock it, people needs >1000 coins or smth to and at least level 200 unlock it permanently for the month of April, as it disapears after that.
Note: under pressure perk won't work on the minigun, because whiloe shooting the player is unable to move. sprinter only works while not shooting the heavy machine gun

**Comment by Gregory-Y** [comment]
There's so many more unique gun ideas though. I thought we all agreed some time ago that the m134 was like the least original concept. Also I would say give it no falloff and nerf the damage to like 9 so that its a consistent 6-shot, but cannot be made a 5-shot by express ammo because having a 5 shot kill weapon with that fire rate is going to upend the game's balance.

That said I don't want to see this weapon in the game at all, there's plenty of concepts that are more original and can make the game better, the M134 would perhaps be the last thing I would want to add because it would be quite difficult to balance. I'd imagine that with so much ammo in the magazine the stream of bullets would cut through pretty much any cover, so pretty much every bd map would be torn apart. Furthermore, its already difficult for people to run away from enemies who are out-spamming them and an M134 would make this impossible.

**Comment by deferred-impact** [comment]
Yeah, i agree with NOTGregory. We have ideas floating around like "a recoilless crab launcher", "a burst fire gun that charges up a special shot", "a sniper with bouncing bullets", and so on. And you're asking for... LMG, but even more annoying? Come on, man.

And yes, i don't want the game to be literally unplayable for the entire month of April.

> High accuracy makes it weak close up

This is a completely nonsense statement in multiple ways. Can you elaborate?

> To unlock it, people needs >1000 coins or smth to and at least level 200 unlock it permanently for the month of April, as it disapears after that.

Oh good, great. Even more grinding for no good reason.
